services/notifications/pkg/config/defaults/defaultconfig.go

```diff
@@ -31,11 +31,8 @@ func DefaultConfig() *config.Config {
 		WebUIURL: "https://localhost:9200",
 		Notifications: config.Notifications{
 			SMTP: config.SMTP{
-				Host:           "",
-				Port:           1025,
-				Sender:         "ownCloud <noreply@example.com>",
-				Authentication: "none",
-				Encryption:     "none",
+				Sender:     "ownCloud <noreply@example.com>",
```

Comment:
I am aware that this is existing code, but why does this even have defaults? Since a mail with that sender will always fail in a spam-filter, why not remove it and disable smtp sending if unset & smtp is configured? Also I do not really like `none` es encryption default....

Reply:
Good question. It's probably only there to serve as an example. I'll try to remove that as well.

Regarding the `none` default value. See also: https://github.com/owncloud/ocis/issues/7356#issuecomment-1735330390 (basically it's really difficult to pick a good default for that)